<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertDialog</class>
 <widget class="QDialog" name="InsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New</string>
  </property>
  <widget class="QPushButton" name="close_pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>30</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="ID_label">
      <property name="text">
       <string>Stuff No.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="ID_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Stuff_label">
      <property name="text">
       <string>Stuff</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Stuff_lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Permission_label">
      <property name="text">
       <string>Permission</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="L1_radioButton">
      <property name="text">
       <string>First Level</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="L2_radioButton">
      <property name="text">
       <string>Second Level</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
